package.json

```diff
@@ -16,7 +16,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@bandwidth/webrtc-node-sdk": "^1.0.0",
+    "@bandwidth/webrtc-node-sdk": "1.0.1",
```

Comment:
So, the node-sdk and browser sdk can be different versions? (I'm new to versioning).

Reply:
Yep! The SDKs are independent.